incubator/tensorflow-serving/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+{{- $gpuCount := .Values.gpuCount -}}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "tensorflow-serving.fullname" . }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app: {{ template "tensorflow-serving.fullname" . }}
+  annotations:
+    "helm.sh/created": {{ .Release.Time.Seconds | quote }}
+spec:
+  replicas: {{ default 1 .Values.replicas }}
```

Comment:
Remove unnecessary `default` function. Defaults should go into `values.yaml` which is already the case here.